From e7df61a1d1f8acfb7d64659a2395283c16452ed5 Mon Sep 17 00:00:00 2001
From: xan <xantares09@hotmail.com>
Date: Sun, 14 Aug 2016 09:02:47 +0200
Subject: [PATCH] Install Python bindings to sitedir

---
 src/Python/CMakeLists.txt | 9 ++++++++-
 1 file changed, 8 insertions(+), 1 deletion(-)

diff --git a/src/Python/CMakeLists.txt b/src/Python/CMakeLists.txt
index 099786d..6cb25f3 100644
--- src/Python/CMakeLists.txt
+++ src/Python/CMakeLists.txt
@@ -107,7 +107,14 @@ IF(OPENMESH_BUILD_PYTHON_BINDINGS)
 
 						ADD_LIBRARY(openmesh SHARED ${SOURCES})
 
-						INSTALL(TARGETS openmesh DESTINATION ${ACG_PROJECT_LIBDIR}/python )
+						if (NOT DEFINED PYTHON_INSTALL_DIR)
+							if (WIN32)
+								set (PYTHON_INSTALL_DIR Lib/site-packages)
+							else ()
+								set (PYTHON_INSTALL_DIR ${ACG_PROJECT_LIBDIR}/python${PYTHON_VERSION_MAJOR}.${PYTHON_VERSION_MINOR}/site-packages)
+							endif ()
+						endif ()
+						install (TARGETS openmesh DESTINATION ${PYTHON_INSTALL_DIR})
 
 						TARGET_LINK_LIBRARIES(
 							openmesh
-- 
2.10.0

